src/components/site/Images.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const Images = props => {
```

Comment:
If we want to keep this in the design system, You can just lines 8 - 20 directly to a Images.md file and then update the [styleguide.config.js](https://github.com/baltimorecounty/dotgov-components/blob/integration/styleguide.config.js#L46) by adding a section, named "Images" under Site Elements. See [here](https://github.com/baltimorecounty/dotgov-components/blob/integration/styleguide.config.js#L9) for an example.

Reply:
@martypowell , I did understand this part . please help

Reply:
resolved per our conversation available here https://github.com/baltimorecounty/dotgov-components/commit/9077c47f7eca6f4b4296d2ad55a87eb142d71f5d